sdk/network/Azure.ResourceManager.Network/api/Azure.ResourceManager.Network.netstandard2.0.cs

```diff
@@ -140,12 +138,12 @@ public partial class ApplicationSecurityGroupCollection : Azure.ResourceManager.
         System.Collections.Generic.IEnumerator<Azure.ResourceManager.Network.ApplicationSecurityGroupResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.Network.ApplicationSecurityGroupResource>.GetEnumerator() { throw null; }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
     }
-    public partial class ApplicationSecurityGroupData : Azure.ResourceManager.Network.Models.NetworkResourceData
+    public partial class ApplicationSecurityGroupData : Azure.ResourceManager.Network.Models.NetworkTrackedResourceData
```

Comment:
Renamed `NetworkResourceData` to `NetworkTrackedResourceData` as it contains location and tags. A new `NetworkResourceData` is added which contains only id, name and resourceType. 

The base type of some resources has been changed from `SubResource` to `NetworkResourceData` as they contain not only id, but also name and resourceType.